etcdserver/auth/auth.go

```diff
@@ -51,7 +51,7 @@ var (
 var rootRole = Role{
 	Role: RootRoleName,
 	Permissions: Permissions{
-		KV: rwPermission{
+		KV: RWPermission{
```

Comment:
So that they can be used elsewhere too -- in tests, for one, but also potentially client auth.